src/components/ui/Dropdown/Dropdown.stories.tsx

```
@@ -0,0 +1,60 @@
+                            <path strokeLinecap="round" strokeLinejoin="round" d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5" />
+                        </svg>
+                    </Dropdown.Trigger>
+
+                    <Dropdown.Content
+                        className="shadow-[0_2px_10px] rounded-md mt-3 divide-y divide-solid"
+                    >
+                        {[['Bookmark', '⌘+B'],['New Private Tab', '⌘+P'],['New Tab', '⌘+T'],['Close Tab', '⌘+W'],['Mute Tab', '⌘+M'], []].map((label) => (
+                            <div
+                                className="flex leading-8 px-4 py-1"
+                            >
+                                {label.at(0)}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)